test/hts-precompile/token-transfer/tokenTransferContract.js

```diff
@@ -35,6 +35,27 @@ describe("TokenTransferContract tests", function () {
     signers = await ethers.getSigners();
   });
 
+  it("should NOT be able to use transferFrom on fungible tokens without approval", async function () {
+    const amount = 1;
+    try {
+      const txTransfer = await tokenTransferContract.transferFromPublic(tokenAddress, signers[0].address, signers[1].address, amount, {gasLimit: 1_000_000});
+      await txTransfer.wait();
```

Comment:
Add something to throw or check in the try so make sure the test fails if it doesn't throw

Reply:
Same for other test